contributed_definitions/nyaml/NXsingle_crystal.yaml

```diff
@@ -0,0 +1,31 @@
+category: base
+doc: |
+  Description of a single crystal material or a single crystalline phase in a material.
+type: group
+NXsingle_crystal(NXobject):
+  sample_orientation(NX_FLOAT):
+    unit: NX_ANGLE
+    doc: |
+      This will follow the Busing-Levy convention:
+      W. R. Busing and H. A. Levy (1967). Acta Cryst. 22, 457-464
+    dimensions:
+      rank: 1
+      dim: [[1, 3]]
+  orientation_matrix(NX_FLOAT):
+    doc: |

```

Comment:
My suggestion add the Busing Levy convention as one subcase under NXrotation_set

Reply:
[link](https://github.com/FAIRmat-NFDI/nexus_definitions/blob/2b18f2cbd7efdce201328c71eda7fbe91e06b6c1/contributed_definitions/nyaml/NXrotation_set_parsed.yaml)